python/nx-cugraph/nx_cugraph/convert_matrix.py

```diff
@@ -35,25 +35,35 @@ def from_pandas_edgelist(
 ):
     """cudf.DataFrame inputs also supported; value columns with str is unsuppported."""
     graph_class, inplace = _create_using_class(create_using)
+    # Try to be optimal whether using pandas, cudf, or cudf.pandas
     src_array = df[source].to_numpy()
     dst_array = df[target].to_numpy()
```

Comment:
`copy=False` is currently the default here, but should we add `copy=False` to be clear about potential data movement? I wonder whether pandas will eventually follow numpy 2 semantics and use `copy=None` to only copy if necessary and `copy=False` to raise if a copy is necessary.

Reply:
> should we add copy=False to be clear about potential data movement?

That seems like a good idea to me, unless there's a reason you think why it might be better not to.

Reply:
NumPy 2 now has these semantics: https://numpy.org/doc/stable/numpy_2_0_migration_guide.html#adapting-to-changes-in-the-copy-keyword